packages/rx_bloc_cli/lib/src/models/bundle_generator.dart

```diff
@@ -92,6 +94,10 @@ class BundleGenerator {
       _bundle.files.addAll(_featureOtpBundle.files);
     }
 
+    if (arguments.pinCodeEnabled) {
```

Comment:
Add comment above, to be in line with other feature bundles

Reply:
It is added